static_inventories/p2solo_server.yml

```diff
@@ -0,0 +1,87 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+          ansible_python_interpreter: /usr/bin/env python
+    development:
+      hosts:
+        p2solo:
+          host: reception+172.23.68.168
+          ansible_host: reception+172.23.68.168
+          cloud_flavor: hpc.v1.vm.1-2-20
+          cloud_image: ubuntu-24.04-noble
+          host_networks:
+            - name: vlan1337
+              security_group: "talos_cluster"
+              add_hostname_to_ip_address: true
```

Comment:
Dit is een Frankenstein constructie of een kopieer-plak-fout. Als p2solo een eigen stack is met stack_name, stack_prefix, eigen group_vars en eigen static_inventory, dan moet het ook z'n eigen netwerken met security groups krijgen. Nu is de machine in vlan 1337 geland en dat is dedicated voor de Talos stack.